.github/ISSUE_TEMPLATE/config.yml

```diff
@@ -0,0 +1,5 @@
+blank_issues_enabled: true
+contact_links:
+  - name: Report broader Zulip issues (beyond Zulip-Terminal)
+    about: Includes best-practices for filing bug reports, feature requests, security vulnerabilities and more.
```

Comment:
nit: Would we want to use something other than just "best-practices"?
The linked page does contain links to best practices.
But, this description may make it sound like the best practices are available 1 level deeper instead of 2 levels deeper.
Which may not be obvious since no link in that issues page contains the explicit mention of "best-practices".

Reply:
Sure, how about:
```suggestion
    about: Includes guidance for filing bug reports, feature requests, security vulnerabilities and more.
```

Reply:
I meant something like "Includes links to best-practices".